### ISSUE TITLE
Domain Generalization for Vision-based Driving Trajectory Generation. (arXiv:2109.13858v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#11416]
Domain Generalization for Vision-based Driving Trajectory Generation. (arXiv:2109.13858v1 [cs.CV])<br>
https://ift.tt/39LJC0l<br>
<p>One of the challenges in vision-based driving trajectory generation is dealing with out-of-distribution scenarios. In this paper, we propose a domain generalization method for vision-based driving trajectory generation for autonomous vehicles in urban environments, which can be seen as a solution to extend the Invariant Risk Minimization (IRM) method in complex problems. We leverage an adversarial learning approach to train a trajectory generator as the decoder. Based on the pre-trained decoder, we infer the latent variables corresponding to the trajectories, and pre-train the encoder by regressing the inferred latent variable. Finally, we fix the decoder but fine-tune the encoder with the final trajectory loss. We compare our proposed method with the state-of-the-art trajectory generation method and some recent domain generalization methods on both datasets and simulation, demonstrating that our method has better generalization ability.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
